Lib/test/test_fcntl.py

```diff
@@ -196,12 +196,14 @@ def test_fcntl_f_getpath(self):
         hasattr(fcntl, "F_SETPIPE_SZ") and hasattr(fcntl, "F_GETPIPE_SZ"),
         "F_SETPIPE_SZ and F_GETPIPE_SZ are not available on all platforms.")
     def test_fcntl_f_pipesize(self):
+        resource = import_module('resource')
```

Comment:
Why not just declare import resource from the top of the file?

Reply:
Oh that's right, I'll fix it.